client/src/components/Header.js

```diff
@@ -98,7 +99,38 @@ const HeaderWrapper = styled.div`
   width: 100vw;
 `;
 
-export default ({ authLoading, onMenuClick, isAuthenticated }) => {
+export default ({ session, authLoading, onMenuClick, isAuthenticated, orgname, user}) => {
+  const menu = (
+    
+    <Menu>
+      <Menu.Item>
+        <Link   to="/profile">
```

Comment:
Formattign is waaaaay off, after I get an answer from the above comments, let me know so I can reformat these...

Reply:
Sure go ahead ! all the comments are addressed.

Reply:
The above comments have been addressed, sure thing ! thanks.